### PR TITLE
[feat] Add vln training code

### PR DESCRIPTION
**Pull request by LiuRicky**
## Description

Add VLN training codes

## Motivation

Closes #
Add VLN training codes

## Changes

- add train_starvln.py
- relative files with starvln.py
- modify vlm data preprocessing, add qwen3vl mrope

## Testing

- [x] Local training for N steps without errors
- [x] Benchmark evaluation results (**required** for framework / benchmark changes, see table below)

| Benchmark | Metric | Result |
|-----------|--------|--------|
|  RxR-CE   |    SR    |   55.8     |

- **Checkpoint**: 
- **Config**: 
- **Reproduce**: 

## Type of Change
- [ ] Bug fix (non-breaking)
- [x] New feature (non-breaking)
- [ ] New framework / benchmark integration
- [ ] Breaking change
- [ ] Documentation only
- [ ] Refactor (no behavior change)

## Checklist

- [x] No unrelated changes mixed in
- [x] New features have example config in `examples/`
- [x] No secrets, API keys, or large binaries committed
- [ ] Files stay isolated — no unnecessary modification to shared modules (`starVLA/dataloader/`, `starVLA/training/`, `starVLA/config/`)
- [ ] No modifications to shared files, or justified above
- [ ] If adding framework/benchmark: checkpoint uploaded to personal HF (public)

### CODE REVIEW

**Comment by LiuRicky** [comment]
@claude detect whether conflict to the merge

**Review by JinhuiYE — APPROVED** [review]
